code/game/machinery/_machinery.dm

```diff
@@ -151,7 +151,7 @@ Class Procs:
 
 	if(!speed_process && init_process)
 		START_PROCESSING(SSmachines, src)
-	else
+	else if(speed_process) // Required to prevent non-speed non-init machines from running
```

Comment:
how about...
```suggestion
	if(init_process)
		if(speed_process) // Required to prevent non-speed non-init machines from running	
			START_PROCESSING(SSfastprocess, src)
		else
			START_PROCESSING(SSmachines, src)
```

Reply:
Also do not just click commit suggestion, this might break it

Reply:
That method makes more sense, but causes mineral machines to stop working.
Found in: `/code/modules/mining/machine_processing.dm`

The Ore Redemption Machine will not accept ores, and the Bluespace Mining Machine doesn't enable after receiving a silo link. There are likely other issues, but those are the two machines I tested.

Reply:
Changing mineral's init_process to TRUE causes the two tested machines to operate normally, but could have unexpected consequences.

The init_process variable was added in the same commit that updated mineral machines to use speed_process and init_process. Commit found here: https://github.com/Sandstorm-Station/Sandstorm-Station-13/commit/f5ee74803f6d1272c61b778344ee38c655bc9e7a

Reply:
just give them the proper required variables to work as ever.

Reply:
If anything, they were not intended to have it at all

Reply:
> If anything, they were not intended to have it at all

This is most likely the case.

Reply:
Let's just keep it off, if anything it's better this way, i'm definitely sure it's unexpected...